pom.xml

```diff
@@ -243,6 +243,11 @@
     	<artifactId>mockito-core</artifactId>
     	<version>2.23.0</version>
     </dependency>
+    <dependency>
+    	<groupId>com.google.code.gson</groupId>
```

Comment:
gson can parse serializable objects and create JSON string automatically, while with org.json you have to build up a JSON object manually. And backwards, you can pass a BufferedReader to the gson while with org.json, you have to read the JSON string first, that can be ineffective in case of large JSON files. (I'm not sure, if we ever will parse JSON files)

Reply:
This seems like a good justification. If you convert all json generation (if any left) in the project to gson, then I write a PMD rule to whine for use of org.json (//ImportDeclaration/Name[contains(@Image,'org.json')]).

Reply:
Do we need that custom List implementation of the Map Choices in JSON?

Reply:
As we have already discussed, that does not have to be custom.